developmentplan.md

```diff
@@ -0,0 +1,15 @@
+![buildah logo](https://cdn.rawgit.com/projectatomic/buildah/master/logos/buildah-logo_large.png)
+
+# Development Plan
+
+## Development goals for Buildah
+
+ *  Integration into OpenShift and potentially other tools.  The biggest requirement for this is to be able build OCI Images without running as root on a system, actually no SYS_ADMIN privs.  This would allow Buildah to run non-privileged containers inside of Kubernetes, so you could distribute your container workloads.
```

Comment:
The goal for OpenShift is not No-Rooot, it is no SYS_ADMIN, which means limited root, or running buildah in a standard linux container.  IE Not modifying any security constraints.   This requirement should mention Kubernetes not OpenShift.

Reply:
I hacked this up a bit.  Please correct if it needs more hacking.